plugins/unmanaged_files/unmanaged_files_inspector.rb

```diff
@@ -279,6 +263,42 @@ def helper_usable?(helper, options)
     false
   end
 
+  def run_helper_inspection(helper, filter, do_extract, file_store_tmp, file_store_final, scope)
```

Comment:
Assignment Branch Condition size for run_helper_inspection is too high. [36.07/15]